<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataPointmap_TINManager_RemoveTrianglesSettingDialog</class>
 <widget class="QDialog" name="GeoDataPointmap_TINManager_RemoveTrianglesSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting for removing triangles with long edges</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="enableCheckBox">
     <property name="text">
      <string>Enable removing triangles with long edges</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="thresholdLabel">
       <property name="text">
        <string>Threshold edge length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="thresholdSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>999.999000000000024</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="note1Label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note&lt;/span&gt;: When &amp;quot;Preview&amp;quot; button is click, the triangles that will be removed are shown painted gray.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="previewButton">
       <property name="text">
        <string>&amp;Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDataPointmap_TINManager_RemoveTrianglesSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDataPointmap_TINManager_RemoveTrianglesSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>thresholdSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
